2023 Q1 Tweaks (#34)
InfosecForActivistsTeam/infosec-activists: content/en/_index.md

will want to turn the GPS receiver off, which prevents from the location data from being
recorded at all. 

> **Bluetooth, WiFi, and Ultrawideband (UWB)** are common technologies that use short-range radios to communicate with

Author: Mentioned UWB here since that is a wireless technology present in the last several generations of phones.

I'd be open to removing this piece since it's not explicitly mentioned within the OS, but it is toggled by the Airplane mode functionality.

Reviewer: Excellent. Anyone with this degree of concern should know that this technology *exists* and is actively in use.

> other devices. Unfortunately, their operation requires your phone to broadcast information which can uniquely itself. This information can be observed by a "beacon" or scanning device.

Reviewer: uniquely what itself
```suggestion
other devices. Unfortunately, their operation requires your phone to broadcast information which can uniquely identify itself. This information can be observed by a "beacon" or scanning device.
```

Stores, government agencies, and police use these devices to identify people who are nearby.
This type of surveillance is profoundly dangerous at an action, so these radios should be disabled
whenever possible. 

**Your mobile carrier** can also track your location using your phone's contact with its cellular towers.
Even if you are not actively in a call, texting, or browsing the internet over a celluar connection, there
is still constant activity between your device and the towers to maintain a connection. These connections
are sufficiently accurate to place an individual on the same block as an action.
This location information is typically logged by your carrier, and can be accessible to law

> enforcement. [Some police departments](https://www.aclu.org/issues/privacy-technology/surveillance-technologies/stingray-tracking-devices-whos-got-them) also have access to devices known

Author: It's an older article, but I haven't seen any updates from EFF. Considering it still relevant since there hasn't been any explicit ban on Stingray devices, and they've gone through several generations at this point.

Reviewer: 👍

> as cell site simulators, IMSI catchers, or a "Stingray". These devices can mimic a cellular tower,

Reviewer: the names changing plurality is a tad strange but I see that as the better side of making the sentence more contrived to avoid it. This wording isn't perfect grammar but ease of understanding is more important, so I approve

Reviewer: Though if anyone has a better sentence suggestion, it could be helpful! @drewbjorn

Author: > the names changing plurality is a tad strange but I see that as the better side of making the sentence more contrived to avoid it. This wording isn't perfect grammar but ease of understanding is more important, so I approve

Agreed- it was weird and I couldn't find a good rewrite after a lot of linguistic gymnastics. If someone has a better wording, I'm all for it!

> as cell site simulators, IMSI catchers, or a "Stingray". These devices can mimic a cellular tower,

Reviewer: "cellular" is a distinctly American term. "Cell phone" does not cross oceans. Where this specific use of the word is about towers, which are already more of a conceptual stretch for a "normal person," I recommend we generalize the phrasing here.
```suggestion
as cell site simulators, IMSI catchers, or a "Stingray". These devices can mimic a mobile phone tower,
```

> and provide the police with not only more accurate, but real-time, data.

Reviewer: ```suggestion
and provide the police with more accurate, real-time data.
```

Both iOS and Android phones have a feature named **"airplane mode,"** to quickly disable the cellular

> radio and some, or all, of the other radios present on the device. The exact behavior is dependent on

Author: iOS doesn't disable _all_ radios initially by default, and the user need to manually disable Bluetooth for the device to disable it the next time Airplane mode is used. The intent here was to explicitly call out this behavior to the reader.

enabled, your phone could leak personally identifiable information (PII) such as your phone number
and email address to [nearby devices](https://arstechnica.com/gadgets/2021/04/apples-airdrop-leaks-users-pii-and-theres-not-much-they-can-do-about-it/).  
 
We recommend keeping AirDrop disabled at all times when near or at an action, and in everyday life

> unless you are actively sharing files with someone. 

Author: No content changes, just whitespace

Apple has prioritized personal security in recent years and now offers features to make securing
your iPhone easier. Following these steps will make your phone reasonably secure for an action. 

If your device does not support *Face ID* as a biometric login method, you will see *Touch ID* on your

> device instead of *Face ID* where it is referenced below.

Author: There is only a single device in Apple's supported devices which has Touch ID so Face ID should be the primary biometric method.

Reviewer: Remember activists are often working with many-generations-outdated tech, including phones that have long fell off the manufacturer's support lists. App Inventor saw OVER TEN year old android phones in regular use in their telemetry.

Author: Resolved via parallel conversation- the text as-is in this PR is fine.

* Set AirDrop to [Receiving Off](https://support.apple.com/en-us/HT204144#setoptions)

> 	* *Settings → General → AirDrop*, tap on **Receiving Off**

Author: Use "tap" vs "select" to be more in-line with the interaction mechanics

* Disable your GPS receiver by turning off Location Services.
	* *Settings → Privacy → Location Services*, toggle *Location Services* off.
	* Even in Airplane mode, iOS will keep the GPS receiver active unless Location Services is turned off.
* Turn off WiFi, and the cellular radio by activating [Airplane Mode](https://support.apple.com/en-us/HT204234)

> 	* Bluetooth **will not** be disabled the first time you use Airplace Mode. If you disable

Author: Add a note about the bluetooth behavior! Bluetooth is no longer disabled in airplane mode unless the user explicitly disables it.

		Bluetooth *after* enabling Airplane mode, iOS will remember this, and disable
		Bluetooth automatically the next time Airplane mode is used.
* Disable your Bluetooth radio

> 	* *Settings → Bluetooth*, toggle the *Bluetooth* slider off.

Author: Keep "toggle" (and update the "tap" wording above), or move this to "tap"? I dislike using two different phrases for similar actions.

* Lock screen setup
	* Disable biometric login
		* *Settings → Face ID & Passcode*, under the section “Use Face ID For:”, toggle all
		  the sliders to off.
	* Set your screen to auto-lock as quickly as possible, and require your passcode immediately.
		* *Settings → Display & Brightness → Auto-Lock*, set to 30 seconds or 1 minute at the most
		* *Settings → Face ID & Passcode*, set “Require Passcode” to “Immediately”

> 	* Practice with iOS’ [Emergency SOS](https://support.apple.com/en-us/HT208076) mode, and

Author: Link to the emergency SOS docs
